src/components/CircularProgress/CircularProgress.js

```diff
@@ -9,17 +9,28 @@ import PropTypes from 'prop-types';
  * import { CircularProgress } from 'weaveworks-ui-components'
  *
  * React.render(
- *   <CircularProgress size={45} />
+ *   <div>
+ *     <CircularProgress size={35} />
+ *     <CircularProgress inline /> Inline
+ *     <CircularProgress center />
+ *   </div>
  * );
  * ```
  */
 class CircularProgress extends React.Component {
   render() {
-    const {size} = this.props;
+    const {size, inline, center} = this.props;
     const styles = {
       width: size,
       height: size
     };
+    if (inline) {
+      styles.display = 'inline-block';
+      styles.verticalAlign = 'middle';
+    }
+    if (center) {
+      styles.margin = '0 auto';
```

Comment:
Maybe it'd set this rather as `marginLeft: auto` and `marginRight: auto`, not to impose anything on the vertical margins.